dask/dataframe/io/tests/test_csv.py

```diff
@@ -885,16 +884,10 @@ def test_csv_with_integer_names():
         assert list(df.columns) == [0, 1]
 
 
-@pytest.mark.slow
 def test_read_csv_of_modified_file_has_different_name():
```

Comment:
I don't think the change to this test captures the spirit of the original. What was the motivation for the change?

Reply:
The original test's intention was that the file changed, so the token should change. But the token only changed because the header changed - not the intent, I believe. My change here replicates that behaviour, but I don't actually thing it's a useful test.

Reply:
Ah I agree that's no useful. So if a file changes do you not expect the token to change?

Reply:
Removed test following discussion.
We *may* at some point choose to make use of fsspec's `ukey` method to tokenise files-on-disk, but we don't call this anywhere yet, and it would come at some cost in the client (because, by design, we are assuming the files list cache might be outdated).